Procfile

```diff
@@ -1,2 +1,2 @@
-web:    cd apps/$GROUPAROO_MONOREPO_APP && WEB_SERVER=true  SCHEDULER=false WORKERS=0 npm start
```

Comment:
This is the one that threw me. Now I'm digging through the website and see we have this pattern in our other deployment docs.